getStaticProps.js

```diff
@@ -16,6 +16,11 @@ export default function createStaticProps(swingsetOptions = {}) {
       const { content, data } = matter(
         fs.readFileSync(component.docsPath, 'utf8')
       )
+      //  Read and parse the component's package.json, if possible
+      const pathToPackageJson = component.path + '/package.json'
```

Comment:
i'm not sure it's entirely necessary here but whenever i'm constructing a path, particularly when i'm merging it with a preset value (from user input or a variable), i like to use `path.join()`. it can normalize out some unexpected weirdness if component.path were to have unexpected results. 

i didn't do a deep dive to ensure we can totally trust `component.path` - it's likely you can, but still felt worth sharing the tip. 

https://nodejs.org/api/path.html#path_path_join_paths

Reply:
Great call, updated to use `path.join()` 👍